### ISSUE TITLE
Cache cleaning slow

### ISSUE THREAD

**Issue by ciyaci1470** [d4rken-org/sdmaid-se#594]
As title says, I'm experiencing very slow performance when I try to clean my apps cache. Cleaning takes much more time than SD Maid Legacy (10-15 seconds at best with it. Letting the app work in background with screen turned off is even worse, it takes 10+ minutes to complete the clean, and I'm not using many apps on my phone. The app doesn't crash or report error logs, I'm running a Cubot Note 20 Pro (8GB) with Android 10. Everything is updated, with the last version of play services installed.

**Comment by d4rken** [comment]
Can you provide a video? Maybe a debug log?

> As title says, I'm experiencing very slow performance with SD Maid SE v0.11.0-beta0 when I try to clean my apps cache. Cleaning takes much more time than SD Maid Legacy (10-15 seconds at best with it, including system apps).

Are you using the accessibility service option in SD Maid 1 too?
It being "slow" is what it would look like if SD Maid can't find the right buttons to click and keeps retrying. Normally it should take 1-3 seconds per app to click all the buttons.

> Letting the app work in background with screen turned off is even worse, it takes 10+ minutes to complete the clean, and I'm not using many apps on my phone. The app doesn't crash or report error logs, I'm running a Cubot Note 20 Pro (8GB) with Android 10. Everything is updated, with the last version of play services installed.

When the screen is off, then no buttons are visible and the lockscreen is probably active too, it is not possible for SD Maid (neither 1 nor 2) to click any "clear cache" buttons then. The screen has to stay on and visible.

**Comment by ciyaci1470** [comment]
Yes, I use the service in both the legacy and SE versions, I'll post a video as soon as I get back from work.

**Comment by ciyaci1470** [comment]
https://github.com/d4rken-org/sdmaid-se/assets/142627803/1bd788d7-ad17-451b-b59c-780d4c6ab5ff


https://github.com/d4rken-org/sdmaid-se/assets/142627803/e2553ce1-b35a-46dd-b6c9-feceafc5be02

**Comment by ciyaci1470** [comment]
I did not turn on developer mode in SD Maid SE.
